_posts/The importance_of_being_earnest.md

```diff
@@ -0,0 +1,262 @@
+---
+title: The Importance Of Being Earnest
```

Comment:
is the YAML description complete?

Reply:
Yep it has these:

---
#YAML part
layout: post
title: The Importance Of Being Earnest
#categories: [  ]
author: Aditi A Pujar, Divyoj Singh, Bhoomika A Bhat, Akshara Sharma, R Bangari
featured: true
---
Categories is commented out since we dont have enough entries yet, I am setting featured as True for all posts right now.